lib/main/STM32_USB_Device_Library/Class/cdc/src/usbd_cdc_core.c

```diff
@@ -547,7 +547,7 @@ uint8_t  usbd_cdc_Setup (void  *pdev,
       else /* No Data request */
       {
         /* Transfer the command to the interface layer */
-        APP_FOPS.pIf_Ctrl(req->bRequest, NULL, 0);
+        APP_FOPS.pIf_Ctrl(req->bRequest, (uint8_t *)&req->wValue, sizeof(req->wValue));
```

Comment:
I suppose this uses Control EP of CDC Interface?

Reply:
Seems that CDC control is transferred over EP0 ...